app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -133,6 +133,11 @@ open class FenixApplication : Application() {
             false
         }
     }
+
+    override fun onTrimMemory(level: Int) {
+        super.onTrimMemory(level)
+        components.core.sessionManager.onLowMemory()
```

Comment:
Make sure to only call this in the main process (See helper method in this class). Otherwise you will re-create the world in every process that is low on memory. In some processes like the gecko tab process you definitely do not want to (indirectly) re-create a new gecko runtime. :)

Reply:
👍